packages/angular/build/src/tools/vite/plugins/remove-sourcemaps.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import type { Plugin } from 'vite';
+
+export const removeSourceMapsPlugin: Plugin = {
```

Comment:
isn't "remove sourcemaps" is wasteful? meaning the bundler already generated and the cpu/memory resources are wasted. WDYT?